### PR TITLE
Hero section completed except responsive breakpoints yet

### LINKED ISSUE #7: Hero Section

### LINKED ISSUE #27: universal .container variable that's in helper.css, doesn't work
We were using `.container` class for general positions from helper.css in the project. But now, .container class has affected by feature.css, not helper.css. So, the positions of header and hero section doesnt seem like normal.

`Header` kısmı aşağı kaydı, altta kalan `section` ise artık en yukarda görüntüleniyor. Belki de `.container` yapısının başka componentlerde kullanımı buna neden olmuş olabilir, bilemiyorum.

### PR DESCRIPTION
**Pull request by celik-arda**
Closes #7 
Closes #27 

Note : please adjust higher zoom rate your browser for true appearance

############################################

#7 Issue 
1. added images and icons to static folder for hero-section
2. completed regular design of the hero section.  The responsive breakpoints and changes will be added coming soon.

###########################################

#27 Issue 
- added just a one line html tag `<br>` to line-380.  That's a temporary solution now.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -377,6 +377,53 @@
...
+                            </div>
+                            <div class="hero-section-link">
+                                <a href="#" class="learn-more-link">
+                                    <img class="learn-more-icon" src="./src/static/hero_section_assets/learn_more_link_icon.png" alt="play-icon">
+                                    Learn more about Opensea
+                                </a>
+                            </div>
+                        </div>
+        
+                        <!-- Hero Section Right Side - (Main Nft İmage, Artist Info etc.) -->
```
Comment (ziarparvaiz) [inline index.html:407]: This component has a little too much box shadow

File: index.html [hunk 2]
```diff
@@ -377,6 +377,53 @@
         </div>
     </header>
 
+    <br>    <!-- Buradaki <br> kodu, header ve altındaki section (hero) arasındaki kayma problemi için geçici olarak kondu -->
+
+            <!-- (start)  HERO SECTION -->
+            <div class="main-hero-area">
```
Comment (ziarparvaiz) [inline index.html:383]: This component had a photo in the background and other components were visible above it.

**Review by ziarparvaiz — APPROVED** [review]
Generally looks good. There are some missing parts in the responsive part. Ellerine Saglik 👍

File: index.html (hunk 2 above)
Comment (celik-arda) [reply index.html:383]: i know those issues that you inform me. Already i'm gonna update somethings. Thanks a lot for your feedback.

TR:
Bu eksikleri de ilk fırsatta tamamlamayı düşünüyorum, orijinal sitedeki `box-shadow` özelliği çok belirsiz ve basit geldi, biraz doğaçlama yaptım. Sanırım arkaplandaki resmi de statik değil dinamik şekilde yapacağım.
Yorum için teşekkürler ahmet. 
:)

**Review by enesoeztekin — APPROVED** [review]
It looks good to me. It's just needed to add responsive behavior to the hero section.

**Comment by celik-arda** [comment]
> It looks good to me. It's just needed to add responsive behavior to the hero section.

Exactly. i'm gonna add it. Thanks for your comment @enesoeztekin

**Pull request merged**